kong/runloop/handler.lua

```diff
@@ -429,6 +430,17 @@ end
 
 
 local function build_router(version)
+  -- as new_router may be interrupted, and after init_worker we assume
+  -- the ROUTE is never nil, we create an empty router which cannot be
+  -- interrupted and will be replaced by the new_router
+  if version == "init" then
+    local err
+    ROUTER, err = Router.new(EMPTY, ROUTER_CACHE, ROUTER_CACHE_NEG, ROUTER)
+    if not ROUTER then
+      log(ERR, "could not initialize router: ", err)
```

Comment:
Make the log more clear, and distinguished from the creation of a normal router and keep consistency with L420.
```suggestion
      log(ERR, "could not create an empty router: ", err)
```